### PR TITLE
last

### PR DESCRIPTION
**Pull request by nazi-pira**

### CODE REVIEW

**Review by sarianne-peltola — APPROVED** [review]
I very much enjoyed reviewing this! Great job! Just some minor things that I noticed, just a little bit indentation in the html. Some coloring in the css. And perhaps a bit some changes in the js. Nice work :) :)

File: code/script.js [hunk 1]
```diff
@@ -11,80 +16,134 @@ const CHARACTERS = [
...
-    hat: true,
-    smoker: false,
+    headgear: true,
+    necklace: false,
+    earrings: false,
+    phone: false,
+    smokinghabit: false,
+    patterns: false,
+    beard: false,
+    button: false
```
Comment (KarolCislowski) [inline code/script.js:26]: It is nice that you experimented with the data provided by the teachers. 👍🏻

File: code/script.js [hunk 2]
```diff
@@ -247,13 +396,16 @@ const generateBoard = () => {
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+  console.log(secret);
```
[the remark this answers is not in the document]
Comment (KarolCislowski) [reply code/script.js:399]: Yes, remember to always clean your code from all console.log()s and commented out code blocks and tips left for you by teachers 🗡️ ✂️

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
Your code looks great, but to test it I need a link to the hosted version. 🌈 

Please provide it and while doing it make your ReadMe file shine like a diamond 💎

**Comment by nazi-pira** [comment]
console.log deleted and netlify link updated

**Review by KarolCislowski — APPROVED** [review]
Great App 🥇

**Pull request closed without merging**